classes/submissionFile/DAO.inc.php

```diff
@@ -1,29 +1,25 @@
 <?php
-
 /**
- * @file classes/submission/SubmissionFileDAO.inc.php
+ * @file classes/submissionFile/DAO.inc.php
  *
  * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2003-2021 John Willinsky
+ * Copyright (c) 2000-2021 John Willinsky
```

Comment:
Looks like the copyright date got changed here?

Reply:
Bad regex replacing. :(